src/components/widgets/DateTimeWidget.js

```diff
@@ -27,7 +45,7 @@ class DateTimeWidget extends React.Component {
       : moment().add(100, "years");
     return (
       <MuiPickersUtilsProvider
-        utils={MomentUtils}
+        utils={MomentUtils}day
```

Comment:
@bag3557 seems like a typo.. seems like the "day" keyword has been accidentally being pasted here.